### PR TITLE
fix: fix activity buttons spacing

### PR DESCRIPTION
**Pull request by andreasjacobsen93**
The activity buttons broke off into two lines because of the width and the icons was spaced a little weird.

The button now looks as below, which I think is a little nicer.
screenshot 2018-10-28 at 21 39 46

### CODE REVIEW

File: resources/assets/sass/people.scss [hunk 1]
```diff
@@ -628,6 +628,7 @@
       a.edit {
         position: relative;
         top: 1px;
+        margin-right: 5px;
```
Comment (asbiin) [inline resources/assets/sass/people.scss]: Carefull about the orientation here: RTL langages might need a `margin-left` (to be verified)
```suggestion
    @if $htmldir == ltr {
        margin-right: 5px;
    } @else {
        margin-left: 5px;
    }
```

Comment (asbiin) [reply resources/assets/sass/people.scss]: Good question! I don't know yet.
Tip: you can test by adding your langage in (this list )[https://github.com/monicahq/monica/blob/master/app/Helpers/LocaleHelper.php#L72] temporarly.

**Comment by asbiin** [comment]
@andreasjacobsen93 still need to change layout for LTR/RTL. What do you think ?

File: resources/assets/sass/people.scss (hunk 1 above)
Comment (andreasjacobsen93) [reply resources/assets/sass/people.scss]: I think this makes sense.. still need to test it though

**Comment by andreasjacobsen93** [comment]
I think the last change makes sense.. still need to test it tho

**Comment by asbiin** [comment]
@andreasjacobsen93 thank you!

**Pull request merged**